bundles/core/src/main/java/com/adobe/cq/wcm/core/components/config/PageItemsConfig.java

```diff
@@ -0,0 +1,61 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.wcm.core.components.config;
+
+import org.apache.sling.caconfig.annotation.Configuration;
+import org.apache.sling.caconfig.annotation.Property;
+
+/**
+ * Context-aware configuration holding information on items to be included in pages:
+ *      - scripts
+ *      - links
+ *      - meta
+ *  This configuration is meant to be used as a context-aware resource.
+ *  See <a href="https://sling.apache.org/documentation/bundles/context-aware-configuration/context-aware-configuration.html#context-aware-resources">Context-Aware Resources</a>
+ *
+ *  Assumed structure:
+ *
+ *  <pre>
+ *      com.adobe.cq.wcm.core.components.config.PageItemsConfig
+ *          + prefixPath="/some/path"
+ *          - item01
+ *              + element=["link"|"script"|"meta"]
+ *              + location=["header"|"footer"]
+ *              + attributeName="attributeValue"
```

Comment:
I would introduce a node structure: a node called `HtmlAttributes` that holds each attribute as a property:
```
HtmlAttributes
  - src = "path-to-page"
  - type = "text/css"
  ...
```
This way the JCR structure is compatible with the multifield widget, the day we use it for the UI.

Reply:
IMO, it's already compatible, without the intermediary node.

Reply:
I updated the docs in 47cec50 to make it clear which is a node and which are properties

Reply:
I would not mix HTML attributes with the element and location properties. We could also have an `attributes` property of type Array where each item is a name/value keypair. But it would look cluttered and wouldn't be as clear as a node structure.

Reply:
Moved attributes to dedicated node in 47cec50